src/evaluator.ts

```diff
@@ -903,7 +904,7 @@ class Evaluator {
 
     for (let index = 0; index < table.body.length; index++) {
       const row = table.body[index];
-      const compareAssertion = row.test.assert?.find((a) => a.type === 'select-best');
+      const compareAssertion = row.test.assert?.find((a) => a.type === 'select-best') as Assertion;
```

Comment:
There's a sniff here that we should have a `isSelectBest(assertion: Assertion): assertion is SelectBestAssertion` method but that method would feel too narrow. Likely when more `select-*` assertions are added that function